src/widgets/WeatherWidget.css

```diff
@@ -17,7 +17,7 @@
 }
 
 .weatherImage {
-  width: 50%;
+  width: 95px;
```

Comment:
I see your point but I'd much rather have a max value here than any arbitrary size. Everyones screen is different, which is why Reacticz is responsive and widgets can be resized. If you deliberstely make the widget larger you'd expect the image to scale a little bit. If you want to keep it small with white space you can also do that and put some space around the widget. 

There already is an issue filed on that topic because sensor icons take too much space, I'll update the weather widget when I fix it to make sure all images don't get too large.